### ISSUE TITLE
Json type in shema causes Error

### ISSUE THREAD

**Issue by JK2Kgit** [Brendonovich/prisma-client-rust#203]
Schema 
```prisma
generator client {
  provider = "cargo +nightly run --bin prisma_cli --"
  output   = "../../tests/src/prisma.rs"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model test {
  id   String @id
  data Json
}
```
Causes 9 compile errors, with the only difference being the code line numbers
```
error[E0433]: failed to resolve: use of undeclared crate or module `serde_json`
   --> tests/src/prisma.rs:255:61
    |
255 |                     ::prisma_client_rust::PrismaValue::Json(serde_json::to_string(&value).unwrap()),
    |                                                             ^^^^^^^^^^ use of undeclared crate or module `serde_json`
```

dependencies
```toml
[dependencies]
prisma-client-rust = { git = "https://github.com/Brendonovich/prisma-client-rust", default-features = false, features=["postgresql"], rev = "a0fb19d25d6869c6398a338e5228d80c81a8c354"}
serde = {version = "^1", features=["derive"]}
```

Checked on:
- https://github.com/Brendonovich/prisma-client-rust/commits/197-prisma-enum-generation-broken
- version 0.6.3

Possibly fixable by changing code line returned with errors in similar way
```rs
::prisma_client_rust::PrismaValue::Json(prisma_client_rust::serde_json::to_string(&value).unwrap()),
```
Or importing 
```rs

pub mod test {
    use prisma_client_rust::serde_json;
```

Possible workaround for now is adding serde_json to dependencies.

**Comment by Brendonovich** [comment]
Looks like I'm not fully qualifying it, will fix soon.

**Comment by Brendonovich** [comment]
@JK2Kgit Should be fixed by `rev = "37e872b697790b988df3742e900bee1e904da9fb"`

**Comment by JK2Kgit** [comment]
@Brendonovich Yes it is a fix Thanks now i will be waiting for 0.6.4
